myuw/templates/handlebars/card/textbook.html

```diff
@@ -4,31 +4,33 @@
 <div class="card"  data-name="TextbookCard" data-type="card">
     <div class="fade-in">
         <h3>Textbooks</h3>
+        <p class="text-danger"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i>Sorry, there is a problem getting data from University Bookstore. We are working with them to fix the issue. </p>
```

Comment:
The card level error message style-- "An error has occurred and MyUW cannot display textbook information right now. Please try again later."